csrc/ops/utils.cpp

```diff
@@ -47,11 +47,11 @@ TensorView* maybe_broadcast_index_tv(TensorView* t, size_t dim, size_t rank) {
   std::vector<bool> bcast_dims(rank, false);
   // broadcast outter on inp to match rank with other.
   if (dim + 1 < rank) {
-    std::fill(bcast_dims.begin() + dim + 1, bcast_dims.end(), true);
```

Comment:
Should we make the parameter of `maybe_broadcast_index_tv` use `int64_t` for `dim`?